src/ert/shared/share/ert/forward-models/res/script/rms.py

```diff
@@ -131,7 +131,7 @@ def init_seed(self, iens):
                 random.seed()
                 seed = random.randint(0, RMSRun._max_seed)
 
-        self.seed = seed % RMSRun._max_seed
+        self.seed = seed % RMSRun._max_seed  # noqa
```

Comment:
So I guess this formatting (% value) is deprecated. What about just taking all those input strings and run:
`new_format_string = input_string.replace('%d', '{}')`
and just use `format` instead?

Reply:
I thought about such hacks, but I think the real issue is that we are using `%d`. As long as we use that, I think it is fair to demonstrate that by use of the modulo operator.